Realm/Realm.Sync/SyncConfiguration.cs

```diff
@@ -87,11 +109,18 @@ internal override Realm CreateRealm(RealmSchema schema)
 
         private Native.SyncConfiguration ToNative()
         {
+            if (!string.IsNullOrEmpty(TrustedCAPath) &&
+                !File.Exists(TrustedCAPath))
+            {
+                throw new RealmFileNotFoundException($"File not found: {TrustedCAPath}");
```

Comment:
Well... the file's not there, the exception contains "FileNotFound", so I went with it :P Perhaps we can throw the system's `FileNotFoundException`?